bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java

```diff
@@ -318,6 +318,11 @@ private boolean refreshProperties(MiIoBasicDevice device) {
         int maxProperties = device.getDevice().getMaxProperties();
         JsonArray getPropString = new JsonArray();
         for (MiIoBasicChannel miChannel : refreshList) {
+            if (!isLinked(miChannel.getChannel())) {
+                logger.info("Skip refresh of channel {} for {} as it is not linked", miChannel.getChannel(),
+                        getThing().getUID());
```

Comment:
This isn't something that the user is required to be notified about.

```suggestion
                logger.debug("Skip refresh of channel {} for {} as it is not linked", miChannel.getChannel(),
                        getThing().getUID());
```

Reply:
Yes, should have changed that after testing

Signed-off-by: Marcel Verpaalen marcel@verpaalen.com